### ISSUE TITLE
Add basic documentation on how to integrate Bevy into mobile applications

### ISSUE THREAD

**Issue by alice-i-cecile** [bevyengine/bevy#10912]
## How can Bevy's documentation be improved?

The documentation on Bevy on mobile platforms is quite sparse. See https://github.com/bevyengine/bevy/discussions/10900.

We're currently missing directions on how to build using Android Studio, how to integrate with other apps via FFI, or how to call OS-level APIs for things such as notifications.

https://github.com/jinleili/bevy-in-app is a compelling piece of third party documentation / template that we should learn from and integrate into our own documentation.

**Comment by atonamy** [comment]
@alice-i-cecile  btw the provided example has performance issue need Bevy creator’s guideline to improve rendering pipeline when Bevy implemented through `SurfaceView` in Android. I can help with documentation and improved example if someone help to explain render part in Bevy with custom window other than winit.

**Comment by RobertoMaurizzi** [comment]
Isn't "how Bevy integrates with Android" the same as "How wgpu/winit integrates with Android"?

**Comment by alice-i-cecile** [comment]
Yes and no: a holistic guide would be useful, and Bevy is likely to have additional setup steps at some points.

**Comment by autofool** [comment]
This is where I'm stuck. Bevy's example is stuck in old versions for dependencies. If I recreate bare project with android studio, the game crashes without error. It won't even attach debugger.
